src/views/forms/FormTransferTransaction/FormTransferTransactionTs.ts

```diff
@@ -90,15 +99,23 @@ export interface MosaicAttachment {
         FormRow,
         ModalTransactionUriImport,
         TransactionUriDisplay,
+        ProtectedPrivateKeyDisplay,
+        ModalFormProfileUnlock,
     },
     computed: {
         ...mapGetters({
             currentHeight: 'network/currentHeight',
             balanceMosaics: 'mosaic/balanceMosaics',
+            currentRecipient: 'account/currentRecipient',
         }),
     },
 })
 export class FormTransferTransactionTs extends FormTransactionBase {
+    @Prop({
+        default: null,
+    })
+    account: AccountModel | Account;
```

Comment:
Do we use and need this property anywhere? If so name of the property felt a bit ambiguous to me.